src/Menu/Menu.spec.js

```diff
@@ -109,4 +110,116 @@ describe('<Menu />', () => {
       assert.strictEqual(list.hasClass('test-class'), true, 'should have the user class');
     });
   });
+
+  describe('mount', () => {
```

Comment:
Did you noticed those tests `/test/integration/Menu.spec.js`? I'm wondering how much collision we have.